pom.xml

```diff
@@ -22,11 +22,11 @@
   <groupId>io.cdap.cdap</groupId>
   <artifactId>cdap-metrics-writer-extensions</artifactId>
   <name>CDAP Metric Writer Extensions</name>
-  <version>1.0.2</version>
+  <version>1.0.3-SNAPSHOT</version>
   <packaging>pom</packaging>
 
   <properties>
-    <cdap.version>6.7.2</cdap.version>
+    <cdap.version>6.7.3-SNAPSHOT</cdap.version>
```

Comment:
We generally don't update the CDAP dependencies from plugins unless needed.

Reply:
Thanks, removed